packages/turf-flatten/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@turf/flatten",
+  "version": "1.0.0",
+  "description": "Turf.js flatten module for converting multipart geometries to singlepart.",
+  "main": "index.js",
+  "scripts": {
+    "test": "node test.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Turfjs/turf.git"
+  },
+  "keywords": [
+    "turf",
+    "geojson",
+    "geography",
+    "gis",
+    "featurecollection"
+  ],
+  "author": "morganherlocker",
```

Comment:
No offence to Morgan :) but since this is a wrapper of @tmcw's library, I would use his name as author.

`Tom MacWright`